docs/dev/convert_layout.rst

```diff
@@ -218,24 +220,40 @@ Second example is for a lightly-layout sensitive operator - batch normalization.
 
 ConvertLayout pass is extremely easy to use. The pass is not a part of default relay.build pipeline. The intended usage is to call it between the framework-to-relay parser and relay.build module call.
 
+In order to specify the layouts to convert to, we create a mapping of heavily-layout sensitive operators to a list of the desired layouts for that operator.
+
 .. code-block:: python
 
     # TFlite framework to Relay parser - Default layout is NHWC
     mod, params = relay.frontend.from_tflite(tflite_model,
                                              shape_dict=shape_dict,
                                              dtype_dict=dtype_dict)
 
+    # We assume our model's heavily-layout sensitive operators only consist of nn.conv2d
+    desired_layouts = {'nn.conv2d': ['NCHW']}
+
     # Convert the layout to NCHW
     # RemoveUnunsedFunctions is used to clean up the graph.
     seq = tvm.transform.Sequential([relay.transform.RemoveUnusedFunctions(),
-                                      relay.transform.ConvertLayout('NCHW')])
+                                      relay.transform.ConvertLayout(desired_layouts)])
     with relay.transform.PassContext(opt_level=3):
         mod = seq(mod)
 
     # Call relay compilation
     with relay.build_config(opt_level=3):
          graph, lib, params = relay.build(mod, target, params=params)
 
+
+The example above only considers data layout, the kernel layout is automatically converted to one that is supported by TVM. If we wish to also convert to a specific kernel layout this can be done like so:
+
+.. code-block:: python
+
+    desired_layouts = {'nn.conv2d': ['NCHW', 'HWIO']}
+    pass = relay.transform.ConvertLayout(desired_layouts)
+
+
+If we wish to select the default choice for a specific layout then the layout should be declared as "default". In the first example, the kernel layout is implicitly defined as "default".
```

Comment:
1. Better to explain the order. Imagining that a new user wants to use this pass, what function/doc he/she should refer to in order to make this list (e.g., [data layout, kernel layout])? A simple version could be saying conv2d is always [data, kernel] and you can check XXX for others.

2. Better to explicitly write down an example saying that `{'nn.conv2d': ['NCHW', 'default']}` is equivalent to `{'nn.conv2d': ['NCHW', 'OIHW']}`.